buildlib/travis-build

```diff
@@ -20,6 +20,12 @@ cd ../build32
 CC=gcc-6 CFLAGS="-Werror -m32" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
 ninja
 
+# aarch64 build to check compilation on ARM 64bit platform
+cd ../build-aarch64
+export PATH=$PATH:/home/travis/aarch64/bin
+CC=aarch64-linux-gnu-gcc CFLAGS="-Werror -Wno-maybe-uninitialized" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
```

Comment:
This can probably just be:

```
CC=$HOME/aarch64/bin/aarch64-linux-gnu-gcc CFLAGS="-Werror -Wno-maybe-uninitialized" cmake -GNinja .. 
```

At the very least PATH should use $HOME